src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -470,7 +478,7 @@ void RemoveTimeSeriesNameFromMetadata(DocumentsOperationContext ctx, string docI
                         tsNamesList.Add(val);
                     }
 
-                    var location = tsNames.BinarySearch(tsName, StringComparison.Ordinal);
+                    var location = tsNames.BinarySearch(tsName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Here, are we *sure* that the names are sorted using case insensitive manner? I would expect that they are case sensitive here.

Reply:
The time-series `Heartrate` and `HeartRate` must be the same. 
Why would you expect it to be case sensitive? 

anyway it should be consistent with:
https://github.com/ravendb/ravendb/blob/bad6628a9e11862a5158da0eaef9c82818e4a16b/src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs#L1656

Reply:
Okay, that ensures the right behavior